requirements_dev.txt

```diff
@@ -2,3 +2,4 @@ pre-commit~=2.20
 pylint~=2.15
 pytest~=7.2
 pytest-cov~=4.0
+setuptools~=57.4
```

Comment:
Why? We have not needed setuptools until now, why add this in the current PR, and why then in _dev?

Reply:
I have tested around changing different requirements without any success. This didn't worked, so it can be removed again...